extensions/amp-list/0.1/amp-list.js

```diff
@@ -851,29 +851,36 @@ export class AmpList extends AMP.BaseElement {
         (response) => {
           userAssert(
             response,
-            'Error proxying amp-list templates, received no response.'
+            'failed fetching JSON data: XHR Failed fetching ' +
```

Comment:
nit: Please capitalize these error messages

Reply:
Done. Might want to fix https://github.com/ampproject/amphtml/blob/4da8b96882786565d2ae2d2725ed1c6ecb9fb300/src/batched-json.js#L70 as well where I copied these from.

Reply:
Thanks for the heads up!